scripts/image-builder/configure.sh

```diff
@@ -27,6 +27,14 @@ sudo dnf install -y "https://dl.fedoraproject.org/pub/epel/epel-release-latest-$
 sudo dnf install -y mock nginx tomcli parallel
 sudo usermod -a -G mock "$(whoami)"
 
+# Install AWS CLI
+AWSCLI_DIR=$(mktemp -d /tmp/awscli.XXXXXXXX)
+pushd "${AWSCLI_DIR}" &>/dev/null
+curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && \
+    unzip -q awscliv2.zip && sudo ./aws/install --update
```

Comment:
will this  update the aws binary to latest (in case its used in a local dev-env) ?

Reply:
Yes,  this is what the `--update` option is for. 
It does not do anything if the version did not change